pyface/i_image_resource.py

```diff
@@ -139,8 +139,8 @@ def __init__(self, name, search_path=None):
         elif search_path is not None:
             _path = [search_path]
         else:
-            _path = [resource_path()]
-        self.search_path = _path + [resource_module()]
+            _path = []
+        self.search_path = _path + [resource_path(), resource_module()]
```

Comment:
I am not sure about this. Previously by explicitly setting `search_path`, the call stack frame is not used. Here we will always apply it.  It looks to me a violation of user expectation: It is a way to switch off magic when it is not applicable or when it causes problems.

Reply:
I see, also CI is now failing as a result of the latest change.  I will spend more time looking into this for the correct solution